### DISCUSSION TITLE
parsing actions

### DISCUSSION THREAD

**Discussion by b4zz4** [tin2tin/Blender_Screenwriter#43] [Ideas]
Based on the description of script actions. Place the actions in the place the script says

![2021-06-30_10-39_07](https://user-images.githubusercontent.com/643833/123973486-2db19f00-d992-11eb-81e4-f1db40f23d95.jpg)

**Comment by tin2tin** [comment]
I don't know anything about "actions". Are they objects(with an ID)? If not it may be better to add it as notes like the [[Shot:]] note. Ex. [[Action: my_action]] but I guess those actions aren't transferable between objects, you would have to ensure that the object is within the scene first. I don't know if the asset browser is supporting actions too? 

I guess at some point the object assigned to word-stuff could be replaced with some asset browser stuff.

**Comment by b4zz4** [comment]
The "Action Editor" keeps animations of an object or Rig. And then you can use it in the "Nonlinear Animation"
I thought about the description of the [scene in fountian](https://fountain.io/syntax#section-action
), which involves a persecution, something like this: `CHARACTER action` or `@CHARACTER action`.

For example:
~~~
The SHIP floats in space.
~~~

And the SHIP is an object and SHIP_floats is an action. 

I use the name of character_action because all the actions are kept within the same project
